config/eslint.config.js

```diff
@@ -0,0 +1,173 @@
+module.exports = {
+  extends: ['airbnb', 'plugin:prettier/recommended'],
```

Comment:
quelques config recommandées :
```suggestion
  extends: ["airbnb",
    "airbnb-typescript",
    "airbnb/hooks",
    "plugin:@typescript-eslint/recommended",
    "plugin:prettier/recommended",
    "prettier",
    "plugin:react/jsx-runtime",
    "plugin:import/typescript",
],
```